bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/DefaultEntryLogger.java

```diff
@@ -1036,6 +1036,9 @@ public EntryLogMetadata getEntryLogMetadata(long entryLogId, AbstractLogCompacto
         // entry log
         try {
             return extractEntryLogMetadataFromIndex(entryLogId);
+        } catch (FileNotFoundException ex) {
```

Comment:
This exception will be thrown by `extractEntryLogMetadataByScanning`, and I don't think it is necessary to catch the exception here.

Reply:
If  `FileNotFoundException` is caught,  no need to proceed with the `extractEntryLogMetadataByScanning(entryLogId, throttler)` method call.  save unnecessary call.